docs/api/index.md

```diff
@@ -84,26 +84,28 @@ These are common parameters that work on most symbol sources (except `sentry`):
 
 - `filters`: a set of filters to reduce the number of unnecessary hits on a
   symbol server. This configuration key is an object with two keys:
-    - `filetypes`: a list of file types to restrict the server to. Possible
-      values: `pe`, `pdb`, `mach_debug`, `mach_code`, `elf_debug`, `elf_code`,
-      `breakpad`)
-    - `path_patterns`: a list of glob matches that need to be matched on the image
-      name. If the debug image has no name it will never match here.
+  - `filetypes`: a list of file types to restrict the server to. Possible
```

Comment:
I think this screwed up indentation in the generated documentation if only two spaces are used.